MaxText/tests/attention_test.py

```diff
@@ -252,11 +252,26 @@ def setUp(self):
         max_prefill_predict_length=16,
     )
     self.cfg = config
+
+    config_cp = pyconfig.initialize(
+        [sys.argv[0], os.path.join(PKG_DIR, "configs", "base.yml")],
+        per_device_batch_size=1.0,
+        run_name="test",
+        enable_checkpointing=False,
+        max_target_length=128,
+        max_prefill_predict_length=16,
+        ici_context_parallelism = 4, # use context parallelism of 4
+    )
+    self.cfg_cp = config_cp
+
     self.rng = jax.random.PRNGKey(0)
 
     devices_array = maxtext_utils.create_device_mesh(self.cfg)
     self.mesh = Mesh(devices_array, self.cfg.mesh_axes)
 
+    devices_array_cp = maxtext_utils.create_device_mesh(self.cfg_cp)
+    self.mesh_cp = Mesh(devices_array, self.cfg_cp.mesh_axes)
```

Comment:
where is this used ? are we asserting on something ?

Reply:
ah sorry about that - I was reusing `self.mesh` no need to redefine, removed it now - thanks Alex

Reply:
I fixed this now, we do need the separate mesh_cp. thank you Alex for pointing this out!